assets/scss/_registry.scss

```diff
@@ -39,3 +39,21 @@
 .registry-entry {
   @extend .shadow;
 }
+
+// Scroll-to-top button styling
+.scroll-container {
+  text-align: center;
+  margin-top: 20px;
+}
+
+.scroll-btn {
+  background-color: #007bff;
+  color: white;
+  bottom: 40px;
+  right: 40px;
+  display: none;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  z-index: 1000;
+  border-radius: 50%;
+  position: fixed;
+}
```

Comment:
Since these styles are no longer confined to the registry, they should be moved. I think that it's fine to move it to the end of `assets/scss/_styles_project.scss`.